docs/docs/deployment/configuration/authentication/LDAP.md

```diff
@@ -14,4 +14,4 @@ this.
 
 LDAP Authentication is available in our [Enterprise Edition](/deployment/configuration/enterprise-edition.md). Please
 contact us if this is of interest. We also support sync-ing of LDAP groups into
-[Flagsmith RBAC groups](/advanced-use/permissions.md#groups)).
+[Flagsmith RBAC groups](/system-administration/rbac).
```

Comment:
Wonder if we should also link to here: /deployment/configuration/authentication/SAML#group-sync. Also, one for the future perhaps, we should move those docs elsewhere - it's kind of odd to me that SaaS users need to go to the deployment section to look at SAML information.

Reply:
OK yep will make a SAML PR too - youre right it needs moving